src/Pages/PageCollection.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace Pages;
+
+use DB\PageNav;
+use DB\PageIndex;
+use Utils\StringUtils;
+use Pages\BasePage;
+use Pages\AboutMePage;
+use Pages\DefaultPage;
+use Pages\DevPage;
+
+class PageCollection {
```

Comment:
Comment?
 * Keeps track of all Types of Pages
 * Associates a "Page Type" with "Page Data" from `page_index`, looked up by `str | int $slug`
 * Returns the correct instantiated Page for the `page_index.type`

Reply:
[16db72e](https://github.com/cdcline/demo-website/pull/55/commits/16db72e01f1ad69fad177cdfca696829cd84238d)